lib/elixir/lib/file.ex

```diff
@@ -920,17 +920,19 @@ defmodule File do
   end
 
   @doc ~S"""
-  Opens the given `path` according to the given list of modes.
+  Opens the given `path` according to the given list of `modes`.
 
   In order to write and read files, one must use the functions
-  in the `IO` module. By default, a file is opened in binary mode,
+  in the `IO` module. By default, a file is opened in `:binary` mode,
   which requires the functions `IO.binread/2` and `IO.binwrite/2`
   to interact with the file. A developer may pass `:utf8` as an
   option when opening the file and then all other functions from
   `IO` are available, since they work directly with Unicode data.
 
   The allowed modes:
 
+    * `:binary` - the default mode.
```

Comment:
Opens the file in binary mode, disabling special handling of unicode sequences (default). 

Something along this line should be appropriate and sufficient.